src/applications/static-pages/burial-how-do-i-apply-widget/components/App/index.js

```diff
@@ -0,0 +1,96 @@
+
+export const App = ({ loggedIn, toggleLoginModal }) => {
+  const setReturnUrl = () => {
+    sessionStorage.setItem(
+      'authReturnUrl',
+      `${
+        window.location.origin
+      }/burials-and-memorials/application/530/introduction/`,
+    );
+  };
+  return (
+    <section aria-labelledby="section-heading">
+      <h2 id="section-heading">How do I apply?</h2>
+      <va-alert
+        close-btn-aria-label="Close notification"
+        status="info"
+        visible
+        aria-labelledby="alert-heading"
+        aria-describedby="alert-description"
+      >
+        <h3 id="alert-heading" slot="headline">
+          Our online burial form isn’t working right now
+        </h3>
+        <div id="alert-description">
+          {loggedIn ? (
+            <>
+              <p>
+                You can still apply for a VA burial allowance by mail. Download
+                the PDF form we provide on this page.
+              </p>
+              <h4>If you started your form online already</h4>
+              <p>
+                You’ll need to start over using a PDF form. But you can still
+                refer to the information you saved in your online form.
+              </p>
+              <a href="/burials-and-memorials/application/530/introduction">
+                Refer to your saved form
+              </a>
+            </>
+          ) : (
+            <>
+              <p>
+                You can still apply for VA pension benefits by mail, in person
+                at a VA regional office, or with the help of a Veterans Service
+                Officer (VSO) or another accredited representative. Download the
+                PDF form we provide on this page.
+              </p>
+              <h4>If you started your form online already</h4>
+              <p>
+                You’ll need to start over using a PDF form. But you can still
+                sign in to VA.gov to refer to the information you saved in your
+                online form.
+              </p>
+              <p>
+                <strong>Note: </strong>
+                We’ll record the potential start date for your benefits as the
+                date you first saved your online form. You have 1 year from this
+                date to submit your application. If we approve your claim, you
+                may be able to get retroactive payments.
+              </p>
+              <va-button
+                onClick={() => {
+                  setReturnUrl();
+                  toggleLoginModal(true);
+                }}
+                text="Sign in to VA.gov"
+              />
+            </>
+          )}
+        </div>
+      </va-alert>
+    </section>
+  );
+};
+
+App.propTypes = {
+  toggleLoginModal: PropTypes.func.isRequired,
+  loggedIn: PropTypes.bool,
+};
+
+const mapStateToProps = state => ({
+  loggedIn: state?.user?.login?.currentlyLoggedIn || null,
```

Comment:
Can state actually be fasly here?

Reply:
I don't know but if it is we will be returning null.